### DISCUSSION TITLE
@prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.

### DISCUSSION THREAD

**Discussion by jwansoo** [prisma/prisma#18382] [Q&A] [answered]
### Bug description

In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues              09:31:09  
    at new PrismaClient (D:\mastering_nuxt_3\mastering_nuxt_3\node_modules\.pnpm\@prisma+client@4.11.0_prisma@4.11.0\node_modules\.prisma\client\index.js:3:11)
    at file:///D:/mastering_nuxt_3/mastering_nuxt_3/.nuxt/dev/index.mjs:693:16
    at ModuleJob.run (node:internal/modules/esm/module_job:194:25)

An error occurred while running the seed command:
Error: Command failed with exit code 1: tsx prisma/seed.mjs

### How to reproduce

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```
 "devDependencies": {
    "@nuxtjs/supabase": "^0.3.1",
    "@nuxtjs/tailwindcss": "^6.6.0",
    "@pinia/nuxt": "^0.4.7",
    "@prisma/client": "^4.11.0",
    "@tailwindcss/typography": "^0.5.9",
    "@vueuse/core": "^9.13.0",
    "@vueuse/nuxt": "^9.13.0",
    "nuxt": "^3.3.1",
    "pinia": "^2.0.33",
    "prisma": "^4.11.0",
    "vite-plugin-vsharp": "^1.4.2"
  },
  "dependencies": {
    "nuxi": "^3.3.1"
  }

**Comment by nurul3101** [comment]
Hello @jwansoo 👋 

Can you please elaborate on how exactly you got this error?
Can you share your seed file?

**Reply by nurul3101** [reply]
@jwansoo Do let us know in case you are still running into this issue

**Comment by rogadev** [comment]
I can't even get my prisma schema to find my DATABASE_URL from the .env file. I've even put the .env inside my prisma folder with only that connection string key/value pair and all it wants to do is connect to localhost:5432.

```bash
$ npx prisma db push
Environment variables loaded from .env
Environment variables loaded from prisma\.env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": PostgreSQL database "mydb", schema "public" at "localhost:5432"

Error: P1001: Can't reach database server at `localhost`:`5432`

Please make sure your database server is running at `localhost`:`5432`.
```

Unless I hardcode the connection string into my prisma.schema I can't get it to use the actual connection string.

**Reply by rogadev** [reply]
Oh, I have to name the runtime config the same name as the env variable.

```env
runtimeConfig: {
  BaseURL: process.env.BASE_URL,
  OpenAI_API_KEY: process.env.OPENAI_API_KEY,
  DATABASE_URL: process.env.DATABASE_URL,
}
  ```

**Reply by nurul3101** [reply]
@rogadev You aren't facing issues anymore, right?

**Comment by jwansoo** [comment, marked as answer]
I think it's a conflict between programs in the package.

I created a new fold, reinstalled the program in the package, moved
all the source code and rerun it and it works fine.


2023년 3월 27일 (월) 오후 11:55, Nurul Sundarani ***@***.***>님이 작성:

**Reply by nurul3101** [reply]
Thanks for getting back and letting us know.
